msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -93,10 +102,19 @@ public abstract partial class ClientApplicationBase
 
             RequestContext requestContext = new RequestContext(new MsalLogger(Guid.Empty, null));
 
+            var platformInformation = new PlatformInformation();
+
```

Comment:
is there a reason why you can't use the PlatformInformation from line 91?

Reply:
fixed